pages/1.10/security/ent/tls-ssl/ca-custom/index.md

```diff
@@ -8,26 +8,317 @@ excerpt:
 enterprise: true
 ---
 
-Each DC/OS cluster has its own DC/OS certificate authority (CA). By default, that CA uses an automatically generated globally unique root CA certificate for signing other certificates. In lieu of using the auto-generated root CA certificate, you can configure DC/OS Enterprise to use a custom CA certificate, which can be a root CA certificate or an intermediate CA certificate.
+# Motivation
+Each DC/OS Enterprise cluster has its own DC/OS certificate authority (CA). By default, that CA uses a globally unique root CA certificate generated during the installation of DC/OS. That root CA certificate is used for signing certificates for the components of DC/OS, such as Admin Router. In lieu of using the auto-generated root CA certificate, you can configure DC/OS Enterprise to use a custom CA certificate, which is *either* a root CA certificate *or* an intermediate CA certificate. (see examples [below](#Example-use-cases))
 
 The benefits of using a custom CA certificate for your DC/OS Enterprise cluster include:
 
-- Ensuring that all X.509 certificates used within the DC/OS cluster (for both signing and encrypting) derive from your organization's X.509 certification hierarchy.
-- Controlling security properties of the key pair (such as type and strength) used for signing DC/OS component certificates.
-- Ensuring that all DC/OS components (including Admin Router) present browser-trusted certificates.
+- Ensuring that all X.509 certificates used within the DC/OS cluster (for both signing and encrypting) derive from your organization’s X.509 certification hierarchy
+- Controlling security properties of the key pair (such as type and strength) used for signing DC/OS component certificates
+- Ensuring that all DC/OS components (including Admin Router) present browser-trusted certificates
 
-Custom CA certificate support is enabled by three install time [configuration parameters](/1.10/installing/ent/custom/configuration/configuration-parameters/#ca-certificate-path-enterprise):
+# The structure of this document page
+To facilitate the the reading of this page we start out by providing a Glossary for general definition of terms, followed by an in-depth configuration parameter reference. An installation walkthrough is provided in section Installing DC/OS Enterprise with a custom CA certificate. Section Example use cases then provides example file contents for the custom CA certificate configuration files for three popular use cases.
```

Comment:
`To facilitate the the` - remove one `the`.

Reply:
`a Glossary`, should be `a glossary` (no caps)

Reply:
Ca we make `Installing DC/OS Enterprise with a custom CA certificate` link to the section?